server/neptune/workflows/internal/root/deployment.go

```diff
@@ -0,0 +1,13 @@
+package root
+
+import (
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/github"
+)
+
+type DeploymentInfo struct {
```

Comment:
I would encourage adding a field to represent version in this struct.  This will ensure if we need to make a backwards compatible change, we can increment that version and keep our parsers backwards compatible.